interface/forms/gad7/gad7.inc.php

```diff
@@ -14,6 +14,8 @@
 
  require_once("../../globals.php");
  require_once("$srcdir/api.inc");
+ require_once("$srcdir/patient.inc");
+ /* require_once("$srcdir/globals.inc.php"); */
```

Comment:
thanks for re-running the tests @stephenwaite .

the change to gad7.inc.php had been made on the branch 'donate_to_OpenEMR', i just did a PR from there and it went as PR request 5209 which is now failing with another, maybe rogue, error 

> Run source ci/ciLibrary.source
> Could not open input file: /var/www/localhost/htdocs/openemr/vendor/bin/phpunit
> Error: Process completed with exit code 1.

i have two branches for different pieces of development, but i could make the change to gad7.in.php on the 'report' branch as well if that's easier? 

also, in the PR for the phq9 form, #5562 - there is an error for a file that i don't think was changed for phq9. I've asked mark sarnov to correct the complained of file though and try another PR.

thanks for your help, as always

Reply:
hi @stephenwaite 
  i think i need help with the correct use of git. On my local machine i have the master and three branches -  'donate_to_OpenEMR' is for the development of the gad7 form, 'reports' is for the development of the latest-encounter report, and then I have a third branch for local changes that won't be submitted to OpenEMR. The first two of these branches are also in my github repo ruthkonyn/openemr (plus the master of course). 

the problem is that the 'report' branch contains an outdated version of the gad7 form, as does 'master'

so when i did a PR request from the report branch, to submit our new report, it also pushed this older version of the gad7 form. In a way all i really wanted to do was submit the latest-encounter report code. 

hence the muddle, and the files in this PR that have not been updated.

is there a good reference that explains how to manage this kind of situation ?

i am presuming that once code has been merged, i.e. accepted, it will then appear in my master branch when i update my local repository, and i can use my branches for further development. Or ... ??? Meanwhile should i be keeping my two development branches in step with each other, or in fact just have a single development branch for all pieces of development work? 

thanks
ruth

Reply:
it's easiest when you create a new branch for a PR from an updated master, recommend closing this and starting a new PR with only the files you wish to contribute